graph/loaders/assets.js

```diff
@@ -20,6 +20,17 @@ const genAssetsByID = (context, ids) => AssetModel.find({
 }).then(util.singleJoinBy(ids, 'id'));
 
 /**
+ * [getAssetsByQuery description]
+ * @param  {Object} context  the context of the request
+ * @param  {String} value    text string to search agains the documents
+ * @param  {Number} limit    limit the number of results
+ * @return {Promise}         resolves the assets
+ */
+const getAssetsByQuery = (context, value, limit) => {
```

Comment:
I suggest putting the query variables into an object, so that it reflects what was defined in the schema and would also be more in line with code that has already been written.

e.g. `const getAssetsByQuery = (context, query) => {`